views/admin/data-contributions.html

```diff
@@ -2,70 +2,76 @@
 
 {% block main %}
 
-<table class="data-curation">
-  <thead>
-    <tr>
-      <th>User</th>
-      <th>Trial</th>
-      <th>File / URL</th>
-      <th>Category</th>
-      <th>Date</th>
-      <th class="comments-col">Curator's comments</th>
-    </tr>
-  </thead>
-  <tbody>
-    {% for dataContribution in dataContributions %}
-      {% if dataContribution.approved %}
-        {% set status = "approve" %}
-      {% elif dataContribution.approved === false %}
-        {% set status = "reject" %}
-      {% else %}
-        {% set status = "" %}
-      {% endif %}
+<section class="data-curation">
```

Comment:
Woohoo, going semantic! \o/

Thanks for doing that. In the same time, I'm screaming internally because I had this itch for weeks at least :sweat_smile: